assets/less/formstyles.less

```diff
@@ -78,25 +78,52 @@ button:focus {
     }
   }
 }
-.btn-primary {
-  background: #2bb673;
-  color: #fff;
-}
-.btn.btn-danger {
-  background-color: #f00;
-  color: #fff;
+.btn-primary, .btn-success {
+  color: @buttonText;
+  background-color: @primaryGreen;
+  border-color: @primaryGreen;
+  &:disabled, &[disabled]:hover {
```

Comment:
Minor but my opinion is using `pointer-events: none;` instead of specifying `&[disabled]:hover` could be a bit cleaner.